understanding/22/target-size-minimum.html

```diff
@@ -57,7 +57,7 @@ <h3>Exceptions</h3>
               <li><strong>Equivalent:</strong> In cases where a target does not have an size equivalent to 24 by 24 CSS pixels, but there is another control that can achieve the underlying function that <em>does</em> meet the minimum size, the target can be excepted based on the "Equivalent" exception.</li>
-              <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, bulleted / numbers lists, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, maintaining a 24 CSS pixels offset between links in adjacent lines of text would require a large line height which can be undesirable in many design contexts. Also, inline numeric footnotes common in scientific texts may sometimes have a size well below 24 CSS pixels. <br/>Note: Stacked links in navigation structure that happen to use list markup do not count as inline links. Authors can anticipate the relative position of these links and provide sufficient target spacing.</li>
+              <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, bulleted or enumerated lists, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, maintaining a 24 CSS pixels offset between links in adjacent lines of text would require a large line height which can be undesirable in many design contexts. The word "enumerated" here refers to list items prefaced by a symbol (whether numbered or lettered) representing an ordered list. Inline numeric footnotes common in scientific texts may sometimes have a size well below 24 CSS pixels. <br/>Note: Stacked links in navigation structure that happen to use list markup do not count as inline links. Authors can anticipate the relative position of these links and provide sufficient target spacing.</li>
```

Comment:
```suggestion
              <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, bulleted or enumerated lists, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, maintaining a 24 CSS pixels offset between links in adjacent lines of text would require a large line height which can be undesirable in many design contexts. The word "enumerated" here refers to list items prefaced by one or more characters (whether numbers or letters) representing an ordered list. Inline numeric footnotes common in scientific texts may sometimes have a size well below 24 CSS pixels. <br/>Note: Stacked links in navigation structure that happen to use list markup do not count as inline links. Authors can anticipate the relative position of these links and provide sufficient target spacing.</li>
```